frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -46,10 +47,12 @@ const GitSection: React.FC = () => {
         />
       )}
       <ExpandCollapse
+        className="odc-expand-collapse"
```

Comment:
The PF3 component is designed this way. It looks like it is also being used by `error.tsx` without any customization.
I suggest we refactor all usages of this component to PF4 or we leave it alone.
Confirm with @spadgett about moving to PF4.

Reply:
My preference would be to move to the PF4 component and update error.tsx as well. The challenge we've had is that the font sizes are so different between PF3 and PF4 that it's tough to mix components. @sg00dwin is trying to address that in #1751 while we migrate to PF4.

We might raise as an upstream Patternfly issue. When everything else is left-aligned, having the left margin in the expand component looks off imo.